tests/frontend/ui/test_home.py

```diff
@@ -92,7 +92,6 @@ def test_add_ons_footer_links(base_url, selenium, i, links):
         'firefox/new',
         'firefox/mobile',
         'firefox/mobile',
```

Comment:
I had to go back to the issue to understand why there were still 2 `'firefox/mobile'` so maybe there should be a comment somewhere in/near the test explaining the links that are being enumerated?

Reply:
I think when I wrote this I was just matching URL's for the pages that loaded.